core-tests/src/test/java/org/mule/runtime/core/routing/IdempotentMessageValidatorTestCase.java

```diff
@@ -7,34 +7,52 @@
 package org.mule.runtime.core.routing;
 
 import static org.hamcrest.Matchers.sameInstance;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertThat;
 import static org.junit.rules.ExpectedException.none;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
+import static org.mule.runtime.api.message.Message.of;
+import static org.mule.runtime.core.el.BindingContextUtils.NULL_BINDING_CONTEXT;
 
 import org.mule.runtime.api.message.Message;
+import org.mule.runtime.api.metadata.TypedValue;
 import org.mule.runtime.core.api.Event;
 import org.mule.runtime.core.api.EventContext;
+import org.mule.runtime.core.api.el.ExpressionLanguageAdaptor;
 import org.mule.runtime.core.api.routing.ValidationException;
+import org.mule.runtime.core.api.util.IOUtils;
+import org.mule.runtime.core.el.DataWeaveExpressionLanguageAdaptor;
 import org.mule.runtime.core.internal.message.InternalMessage;
 import org.mule.tck.core.util.store.InMemoryObjectStore;
 import org.mule.tck.junit4.AbstractMuleContextTestCase;
+import org.mule.weave.v2.el.ByteArrayBasedCursorStreamProvider;
+import org.mule.weave.v2.el.WeaveDefaultExpressionLanguageFactoryService;
 
+import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
 public class IdempotentMessageValidatorTestCase extends AbstractMuleContextTestCase {
 
+  private static IdempotentMessageValidator idempotent = new IdempotentMessageValidator();
```

Comment:
no need to instantiate here, you are already instantiating in the `@Before` method